xml/System/Uri.xml

```diff
@@ -6030,7 +6030,8 @@ The following examples show a URI and the results of calling <xref:System.Uri.Ge
           <format type="text/markdown"><![CDATA[
 
 ## Remarks
- The <xref:System.Uri.UserEscaped%2A> property is set to `true` to indicate that the string used to create the <xref:System.Uri> instance was completely escaped before it was passed to the constructor; that is, the `dontEscape` parameter of the constructor call was set to `true`.
+
+ The <xref:System.Uri.UserEscaped> property was originally designed to indicate that the string used to create the <xref:System.Uri> instance was completely escaped before it was passed to the constructor; that is, the `dontEscape` parameter of the constructor call was set to `true`. However, since the `dontEscape` parameter is now obsolete and always treated as `false`, this property always returns `false`.
```

Comment:
```suggestion
The <xref:System.Uri> constructors that have a `dontEscape` parameter have been obsolete since .NET Framework 2.0 and shouldn't be used. As a result, this property is deprecated and shouldn't be used either.
```